ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/map.view.js

```diff
@@ -260,15 +261,17 @@ module.exports = Marionette.LayoutView.extend({
   saveAsHome: function() {
     const boundingBox = this.map.getBoundingBox()
     const userPreferences = user.get('user').get('preferences')
-    userPreferences.once('sync', () =>
-      announcement.announce({
-        title: 'Success!',
-        message: 'New map home location set.',
-        type: 'success',
-      })
-    )
+    if (!_.isEqual(boundingBox, userPreferences.get('mapHome'))) {
```

Comment:
```suggestion
    saveAsHome: function() {
      const boundingBox = this.map.getBoundingBox()
      const userPreferences = user.get('user').get('preferences')
      userPreferences.set('mapHome', boundingBox)
      announcement.announce({
        title: 'Success!',
        message: 'New map home location set.',
        type: 'success',
      })
  }
```
❗️ In this case it'd be better to always announce and not bother waiting for the save to actually happen.